jqPropertyGrid.js

```diff
@@ -152,6 +152,14 @@
 			if (postCreateInitFuncs) { postCreateInitFuncs.push(initColorPicker(elemId, value, meta.options)); }
 			if (getValueFuncs) { getValueFuncs[name] = function() {return $('#'+elemId).spectrum('get').toHexString();}; }
 
+		// If label (for read-only)
+        	} else if (type === 'label') {
+			if (meta.description !== undefined) {
```

Comment:
Please use (typeof meta.description === 'string' $$ meta.description), we are making sure  meta.description is a non empty string..